gwdatafind/ui.py

```diff
@@ -56,9 +57,15 @@ def connect(host=None, port=None):
         else:
             port = int(port)
     if port not in (None, 80):
-        cert, key = find_credential()
         context = ssl.create_default_context()
-        context.load_cert_chain(cert, key)
+        token = find_scitoken(aud='host', scope='read:/LIGODataFind')
```

Comment:
Is the `LIGO` component of the scope name fixed already? The service is now generically `GWDataFind`, so if we have time to swap `LIGO->GW` that would be nice.